include/OpenSpaceToolkit/Astrodynamics/Trajectory/LocalOrbitalFrameFactory.hpp

```diff
@@ -51,6 +51,16 @@ class LocalOrbitalFrameFactory
     /// @return True if local orbital frame factory is defined
     bool isDefined() const;
 
+    /// @brief Accessor for the parent frame.
+    ///
+    /// @return The parent frame
+    const Shared<const Frame> accessParentFrame() const;
```

Comment:
@phc1990 shouldn't this be this?
`const Shared<const Frame>& accessParentFrame() const;`